cmd/ls.go

```diff
@@ -23,7 +23,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/dustin/go-humanize"
+	humanize "github.com/dustin/go-humanize"
```

Comment:
why is this change needed?

Reply:
That happens automatically I think due to goimports

Reply:
there is a reference to humanize.IBytes to humanize size on line 50,  this was automatically added by go tools installed on the editor.